app/decorators/cd_api/hearing_day_decorator.rb

```diff
@@ -0,0 +1,9 @@
+# frozen_string_literal: true
+
+module CdApi
+  class HearingDayDecorator < BaseDecorator
+    def to_datetime
+      sitting_day&.to_datetime
```

Comment:
Opinion: Does this need to be a decorator for a single method? the testing/complexity vs actually writing this seems overkill.

Reply:
Ah yep, i'm hoping to write that as tech debt.

The HearingsSorter and HearingsDateSorter  is where it expects the hearing_day to be a string that can convert to DateTime.

In V2 it's an object CdApi::HearingDay with an attribute sitting_day. I added a decorator to allow the call #to_datetime on the object and return siting_day in DateTime type because this object just has one attribute sitting_day.

I'll write this up as tech debt to fix. The alternative will bloat this PR more than having the decorator and a tech debt ticket